app/features/my.journey.js

```diff
@@ -0,0 +1,9 @@
+let MyJourney = function(database) {
+    this.database = database;
+};
+
+MyJourney.prototype.now = function(login, callback) {
```

Comment:
Single journey, so that we can overwrite it from the front end and are sure to only ever get the one for the user.

Reply:
Considering also that the Create could first check that the user has only one, but this is currently handled in the front end where we track the current Journey ID.

Reply:
TODO: remove MyJourney*S* and associated code, as well as step table code